about_bootstrap.md

```diff
@@ -0,0 +1,29 @@
+## What you will learn in this section
+This section will basically help you to know what is bootstrap and why you should learn it.
+
+## About bootstrap
+Bootstrap, originally named Twitter Blueprint, was developed by Mark Otto and Jacob Thornton at Twitter as a framework to encourage consistency across internal tools. Here we will talk about why Bootstrap has capturted the developers attraction:
+
+We always think of investment and how to save money, talking of Bootstrap it is free and open-source frontend framework for designing website or web based applications. But I agree there would a question in your mind what does this bootstrap contain and how it is used?
+
+We will learn how it is used in this tutorial but first let me discuss what it contains. Bootstrap contains HTML, CSS based designed template for verious components like button, pagination, progress bar, different icon and Javascript extension.
```

Comment:
This line is not correct - Bootstrap contains HTML, CSS based designed template for verious components like button, pagination, progress bar, different icon and Javascript extension.

Bootstrap doesn't contains, perhaps Bootstrap made on top of HTML5 and CSS3.